client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js

```diff
@@ -0,0 +1,354 @@
+							/>
+						</div>
+						<div className="carrier-accounts__settings-two-columns">
+							<TextField
+								id={ 'postal_code' }
+								title={ translate( 'Postal code / Zip' ) }
+								value={ getValue( 'postal_code' ) }
+								updateValue={ updateValue( 'postal_code' ) }
+								error={ fieldErrors.postal_code }
+							/>
+							<TextField
+								id={ 'phone' }
+								title={ translate( 'Phone' ) }
+								value={ getValue( 'phone' ) }
+								updateValue={ updateValue( 'phone' ) }
+								error={ fieldErrors.phone }
+							/>
+						</div>
+						<TextField
+							id={ 'email' }
+							title={ translate( 'Email' ) }
+							value={ getValue( 'email' ) }
+							updateValue={ updateValue( 'email' ) }
+							error={ fieldErrors.email }
+						/>
+					</CompactCard>
+					<CompactCard className="carrier-accounts__settings-company-info">
+						<div className="carrier-accounts__settings-header">
+							<h4 className="carrier-accounts__settings-subheader">{ translate( 'Company information' ) }</h4>
+							<p className="carrier-accounts__settings-subheader-description">{ translate( 'This is the company info you used to create your UPS account' ) }</p>
+						</div>
+						<TextField
+							id={ 'company_name' }
+							title={ translate( 'Company name' ) }
+							value={ getValue( 'company_name' ) }
+							updateValue={ updateValue( 'company_name' ) }
+							error={ fieldErrors.company_name }
+						/>
+						<div className="carrier-accounts__settings-two-columns">
+							<TextField
+								id={ 'job_title' }
+								title={ translate( 'Job title' ) }
+								value={ getValue( 'job_title' ) }
+								updateValue={ updateValue( 'job_title' ) }
+								error={ fieldErrors.job_title }
+							/>
+							<TextField
+								id={ 'company_website' }
+								title={ translate( 'Company website' ) }
+								value={ getValue( 'company_website' ) }
+								updateValue={ updateValue( 'company_website' ) }
+								error={ fieldErrors.company_website }
+							/>
+						</div>
+					</CompactCard>
+					<CompactCard className="carrier-accounts__settings-ups-info">
+						<div className="carrier-accounts__settings-header">
+							<h4 className="carrier-accounts__settings-subheader">{ translate( 'UPS account information' ) }</h4>
+							<Checkbox id={ 'license_agreement' } checked={ showUPSInvoiceFields } onChange={ updateShowUPSInvoiceFields  } />
+							<span>{ translate( 'I have been issued an invoice from UPS within the past 90 days' ) }</span>
+						</div>
+						{ showUPSInvoiceFields && upsInvoiceFields() }
+					</CompactCard>
+					<CompactCard className="carrier-accounts__settings-license-agreement">
+						<Checkbox id={ 'license_agreement' } checked={ !! getValue( 'license_agreement' ) } onChange={ updateValue( 'license_agreement' )  } />
+						<span>{ translate( 'I have read the {{a}}License Agreement{{/a}}', { components: { a: <a href="https://link.to.terms.com/" />  } } ) }</span>
```

Comment:
Do we have another issue for updating `https://link.to.terms.com`?

Reply:
Apparently we don't need to include that checkbox anymore, so I'm gonna remove it. See: p1592578267297800-slack-fulfill